Sources/JExtractSwiftLib/SwiftTypes/SwiftFunctionSignature.swift

```diff
@@ -195,6 +207,22 @@ extension SwiftFunctionSignature {
     }
     let valueType = try SwiftType(varTypeNode, symbolTable: symbolTable)
 
+    var effectSpecifiers: [SwiftEffectSpecifier]? = nil
+    switch binding.accessorBlock?.accessors {
+    case .getter(let getter):
+      if let getter = getter.as(AccessorDeclSyntax.self) {
+        effectSpecifiers = Self.effectSpecifiers(from: getter)
```

Comment:
`AccessorBlockSyntax.Accessors.getter` can only contain `CodeBlockItemListSyntax` so this should be unreachable.